api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -588,7 +588,7 @@ message Cluster {
 
   // The :ref:`load balancer type <arch_overview_load_balancing_types>` to use
   // when picking a host in the cluster.
-  LbPolicy lb_policy = 6 [(validate.rules).enum = {defined_only: true}];
+  LbPolicy lb_policy = 6 [(validate.rules).enum = {defined_only: true not_in: 7}];
```

Comment:
It's not possible to use named enum values in PGV constraints?

Reply:
Don't think so -- protoc expects an integer value for enum optional values (and PGV handles the vals as ints without name conversions)

Reply:
I should add a comment for what "7" is in that case. I'll guard it with a "comment" annotation